cl/citations/tests.py

```diff
@@ -37,106 +37,97 @@ def test_make_html_from_plain_text(self) -> None:
         """Can we convert the plain text of an opinion into HTML?"""
         # fmt: off
 
-        full_citation_html = ('<pre class="inline">asdf </pre><span class="'
-                              'citation no-link"><span class="volume">22'
-                              '</span> <span class="reporter">U.S.</span> '
-                              '<span class="page">33</span> </span><pre class='
-                              '"inline">asdf</pre>')
         test_pairs = [
             # Simple example for full citations
-            ('asdf 22 U.S. 33 asdf', full_citation_html),
+            ('asdf 22 U.S. 33 asdf',
+             '<pre class="inline">asdf </pre><span class="'
+             'citation no-link">22 U.S. 33</span><pre class="'
+             'inline"> asdf</pre>'),
 
             # Using a variant format for U.S. (Issue #409)
-            ('asdf 22 U. S. 33 asdf', full_citation_html),
```

Comment:
I actually think this was a bug before -- isn't our principle to not change the underlying text of the opinion at all? As this test suggests, we seemed to have been using the erstwhile canonical `reporter` attribute instead of the original `reporter_found` when constructing the citation HTML.

Reply:
Yeah, better not to mess with it, though I could see my past self thinking we could make it better by using the typo-free version of the reporter.